scripts/install-deps.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/sh
 set -eux
 cd "$(dirname "$0")/.."
-bundle install --jobs=4 --retry=3 --path vendor/bundle
+
+bundle install --deployment --without=development --jobs=4 --retry=3 --path vendor/bundle
```

Comment:
Is `--path` vendor/bundle needed since we already included `--deployment`?

re: https://bundler.io/man/bundle-install.1.html#DEPLOYMENT-MODE